bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/HeapSizeManager.java

```diff
@@ -183,5 +188,10 @@ public void onFailure(Throwable t) {
 
   public void markCanBeCompleted(Long id) {
     completedOperationIds.offer(id);
+    if (outstandingOperationsCount.decrementAndGet() == 0) {
```

Comment:
if markCanBeCompleted(x) is called multiple times, wouldn't outstandingOperationsCount get decremented multiple times?